Source/AwsCommonRuntimeKit/mqtt/Mqtt5Client.swift

```diff
@@ -0,0 +1,31 @@
+///  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+///  SPDX-License-Identifier: Apache-2.0.
+
+import Foundation
+import AwsCMqtt
+import AwsCIo
+
+public class Mqtt5Client {
+     private var rawValue: UnsafeMutablePointer<aws_mqtt5_client>
+     private let clientOptions: MqttClientOptions
```

Comment:
I don't think we need this to be a property within the Mqtt5Client. Can we use it in the init and then let it fall out of scope?